app/controllers/internal/profile_fields_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Internal
+  class ProfileFieldsController < Internal::ApplicationController
+    layout "internal"
+
+    def index
+      @profile_fields = ProfileField.all
+    end
+
+    def update
+      @profile_fields = ProfileField.find(params[:id])
+      @profile_fields.update!(profile_field_params)
+      redirect_to internal_profile_fields_path
```

Comment:
*suggestion (non-blocking):** I know this is only a first pass at the admin UI, but can we set a success flash to let the user know the `update` worked (same for `create` and `destroy`).